rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -118,45 +126,44 @@ def __init__(self, config: FCConfig):
         super().__init__()
         self.input_dim = config.input_dim
         self.hidden_layers = config.hidden_layers
-        self.activation = config.activation
-        self.output_activation = config.output_activation
 
+        activation_class = getattr(nn, config.activation, lambda: None)()
```

Comment:
Same here, can we add a TODO that described that FCNet will eventually be retrieved from the ModelCatalog in progress? Just want to make sure we are keeping track of all the moving (and in-the-pipeline) parts right now to keep ourselves and users as best informed as possible about why we are doing all this.

Reply:
added a general todo to the top of the file